### PR TITLE
Release 1.1.0

### LINKED ISSUE #4: Upgrade to support latest (5.4.1) gradle version

### LINKED ISSUE #6: Example tasks to import/export content as MLCP archive
```
task exportContent(type: com.marklogic.gradle.task.MlcpTask) {
  systemProperty "java.library.path", "mlcp/lib/native"
  classpath = configurations.mlcp
  command = "EXPORT"
  port = mlAppConfig.restPort
  output_file_path = "export/" + getDate()
  output_type = "archive"
}

task importContent(type: com.marklogic.gradle.task.MlcpTask) {
  systemProperty "java.library.path", "mlcp/lib/native"
  classpath = configurations.mlcp
  command = "IMPORT"
  port = mlAppConfig.restPort
  input_file_path = "export/"
  input_file_type = "archive"
}

def getDate() {
  return new Date().format('yyyyMMdd-HHmmss')
}

```

### LINKED ISSUE #19: Move over ml-content to ml-data, to follow suite with ml-gradle's mlLoadData task and remove other OBE tasks
Allows dropping mlLoadContent task as well. Use collections.properties to assign a config collection to those docs..

### LINKED ISSUE #20: Load content fails with latest java
You might see something like this using Java 10 or 12:

> Task :mlLoadContent
20/04/16 10:58:29 INFO contentpump.LocalJobRunner: Content type is set to MIXED. The format of the inserted documents will be determined by the MIME type specification configured on MarkLogic Server.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.hadoop.security.authentication.util.KerberosUtil (file:/Users/poneill/.gradle/caches/modules-2/files-2.1/org.apache.hadoop/hadoop-auth/2.6.0/b0b8dec23a84ac8a0d00fbd69a87d320724ae34a/hadoop-auth-2.6.0.jar) to method sun.security.krb5.Config.getInstance()
WARNING: Please consider reporting this to the maintainers of org.apache.hadoop.security.authentication.util.KerberosUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
20/04/16 10:58:30 INFO contentpump.ContentPump: Job name: local_1708497191_1
20/04/16 10:58:30 INFO contentpump.FileAndDirectoryInputFormat: Total input paths to process : 2
20/04/16 10:58:30 ERROR contentpump.LocalJobRunner: Error checking output specification: 
20/04/16 10:58:30 ERROR contentpump.LocalJobRunner: java.lang.IllegalArgumentException: No enum: segment

### PR DESCRIPTION
**Pull request by mariannemyers**
We'll use this PR for any cleanup or small changes needed before we release 1.1.0

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,8 +7,8 @@
  */
 
 plugins {
-  id "net.saliman.properties" version "1.4.6"
-  id "com.marklogic.ml-gradle" version "3.10.1"
+  id "net.saliman.properties" version "1.5.1"
+  id "com.marklogic.ml-gradle" version "3.17.0"
```
Comment (mariannemyers) [inline build.gradle]: Upgrade to 4.0.3.  Test against Java 8 and Java 11.

File: build.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ configurations {
 }
 
 dependencies {
-  mlcp "com.marklogic:mlcp:9.0.7"
+  mlcp "com.marklogic:mlcp:10.0.3"
```
Comment (mariannemyers) [inline build.gradle]: Bump to 10.0.4

File: ml-modules/options/all.xml [hunk 3]
```diff
@@ -18,40 +24,24 @@
...
   <sort-order direction="descending">
     <score/>
   </sort-order>
 
   <!-- Not used, but the default snippeting behavior if no state (see compact/detailed below) was specified -->
   <transform-results apply="snippet">
-    <preferred-elements><element ns="" name="body"/></preferred-elements>
+    <preferred-matches>
+      <element ns="http://marklogic.com/entity-services" name="instance"/>
+      <json-property>instance</json-property>
```
Comment (mariannemyers) [inline ml-modules/options/all.xml]: Default snippeting does not work with a complex object.  `instance` in the DHF schema is always a complex object.  Consider alternatives for how to snippet meaningfully.

**Review by mariannemyers — COMMENTED** [review]
Only Java 9+ is supported by ml-gradle 4.x.  This project will no longer work with Java 8.

File: build.gradle (hunk 1 above)
Comment (mariannemyers) [reply build.gradle]: Bumped in commit f5cf5e1. Tested against Java 11.

File: build.gradle (hunk 2 above)
Comment (mariannemyers) [reply build.gradle]: Bumped in commit f5cf5e1.

File: ml-modules/options/all.xml (hunk 3 above)
Comment (mariannemyers) [reply ml-modules/options/all.xml]: I think this should be its own ticket for future implementation in 2.0.0.

File: src/main/ui-modules/options/all.xml [hunk 4]
```diff
@@ -18,40 +24,24 @@
...
   <sort-order direction="descending">
     <score/>
   </sort-order>
 
   <!-- Not used, but the default snippeting behavior if no state (see compact/detailed below) was specified -->
   <transform-results apply="snippet">
-    <preferred-elements><element ns="" name="body"/></preferred-elements>
+    <preferred-matches>
+      <element ns="http://marklogic.com/entity-services" name="instance"/>
+      <json-property>instance</json-property>
```
Comment (grtjn) [inline src/main/ui-modules/options/all.xml]: It is a separate ticket to look closer at this, but we need to drop the json-property bit at the least. It causes search to fail, which is worse than just showing snippets starting with header data..

**Review by grtjn — COMMENTED** [review]
I think this is good to go once with bypass the snippet issue for json..

File: src/main/ui-modules/options/all.xml (hunk 4 above)
Comment (mariannemyers) [reply src/main/ui-modules/options/all.xml]: Delete      ` <json-property>instance</json-property>`

Comment (mariannemyers) [reply src/main/ui-modules/options/all.xml]: Deleted line.

**Pull request merged**